benchmarks/src/main/java/org/elasticsearch/benchmark/search/aggregations/AggConstructionContentionBenchmark.java

```diff
@@ -252,6 +252,11 @@ public BigArrays bigArrays() {
             return bigArrays;
         }
 
+        @Override
+        public BigArrays bigArraysForResults() {
```

Comment:
I'm not sure this is necessary.  Or, more specifically, what is the use case for big arrays that get cleaned up at the end of the collect phase?  Are there data large enough that we need big arrays for them, but we don't want to pass them onto the reduce?